.changeset/kind-humans-explode.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/data-grid": patch
+---
+
+Fixed the Data Grid crashing when a user scrolls to the bottom and navigates to the last page
```

Comment:
should pagination be mentioned somewhere here?

Reply:
It's sort of inferred when it says "navigates to the last page". You could add "using the pagination" at the end I guess to make it clearer.